uber/models.py

```diff
@@ -904,6 +912,12 @@ def hotel_status(self):
         else:
             return 'Hotel nights: ' + hr.nights_display
 
+class WatchList(MagModel):
+    first_name = Column(UnicodeText)
+    last_name = Column(UnicodeText)
+    disabled = Column(Boolean, default=False)
```

Comment:
> I think disabling watch list items rather than deleting them is good, actually - it means we have the ability to look back through old items to make sure someone's not a repeat offender.

This is definitely a larger discussion we need to have about what it means to delete something.  What I think we agree on is the general use case, "As an admin, I want to be able to always be able to see all of the historical data about someone."  From there it's really just a question of the best way to go about implementing that use case.

My point about setting an ``actually_exists`` field to ``False`` (or whatever) is that in most cases it probably complicates the code too much compared to other ways to accomplish the same use case, though obviously that can vary on a case by case basis.